pnpm-lock.yaml

```diff
@@ -1683,6 +1687,13 @@ packages:
       prettier: 2.8.4
     dev: true
 
+  /@commitlint/types@17.4.4:
```

Comment:
looks like this got added to the lockfile but not to `package.json`

did you use `pnpm add` to add this? try `pnpm add -D -w @commitlint/types`

Reply:
@holic I did use `pnpm add`. I ran the command, and it updated the lock file, but there were no changes in the `package.json` :( Am I missing something?